### PR TITLE
Kc prep js funcs

### PR DESCRIPTION
**Pull request by kev1n80**
+ Make all js functions have params, make it ready to take in data from servlets
+ Added a bio to the user page

Screenshot 2020-07-29 at 5 16 57 PM

### CODE REVIEW

File: src/main/webapp/listing-detailed.js [hunk 1]
```diff
@@ -24,6 +24,50 @@ export default function createListingDetailedView(divCardContainerElement,
...
+      'donations to Fake Address 123, Los Angeles, CA 34567 on weekdays. ' + 
+      'Volunteer at Fake Address 123, Los Angeles, CA 34567 at 8:30PM everyday';
+  divCardInfoElement.appendChild(createListingCardDescription(exComments, 
+      exDescription, exHowToHelp));
+
+  return divCardContainerElement;
+}
+
+/**
+ * Create an element that represenets an exit button for the user to exit the
```
Comment (zrideaux) [inline src/main/webapp/listing-detailed.js:63]: Represents is misspelled here

Comment (kev1n80) [reply src/main/webapp/listing-detailed.js:63]: LOL smh at myself. Nice catch!

File: src/main/webapp/listing-detailed.js [hunk 2]
```diff
@@ -33,75 +77,74 @@ export default function createListingDetailedView(divCardContainerElement,
...
+ * @param name the name of this listing
+ * @param upvotes the number of upvotes this listing has
+ * @param views the number of views this listing has received
+ * @param websiteLink the link to this listings website
  * @return a div with the picture, name, category, reputation, listing details 
  *     (see below) and website of a listing.
  */
-function createListingCardInformation() {
+function createListingCardInformation(category, date, imgSrc, name, upvotes, 
+    views, websiteLink) {
```
Comment (zrideaux) [inline src/main/webapp/listing-detailed.js:109]: Will we want to / need to format integer properties (e.g. upvotes and views) before creating elements with them?

ex. 1234 -> 1,234

Comment (kev1n80) [reply src/main/webapp/listing-detailed.js:109]: Yup, sorry I forgot to ask you all. We should talk about this in the group chat

**Pull request merged**